node/pkg/governor/governor.go

```diff
@@ -686,16 +686,8 @@ func (gov *ChainGovernor) CheckPendingForTime(now time.Time) ([]*common.MessageP
 			foundOne := false
 			prevTotalValue, err := gov.TrimAndSumValueForChain(ce, startTime)
 			if err != nil {
```

Comment:
This error statement is now impossible to hit, right? `TrimAndSumValueForChain` will always return a value instead of an error.

Reply:
It's possible, there's another error case for that function relating to overflow/underflow for int64 calculations. Much, much less likely though.